### ISSUE TITLE
Arguments not getting parsed!

### ISSUE THREAD

**Issue by ahmadawais** [thelowlypeon/cloudflare-cli#2]
Arguments are not getting parsed.

```bash
cfcli zone -s"site.com"

```

Doesn't work it says `Cloudflare site is required`

**Comment by thelowlypeon** [comment]
hi @ahmadawais, it should work if you have a space after the `-s` flag:

```bash
cfcli zone -s "site.com"
```

does that work?

**Comment by ahmadawais** [comment]
Yeah, that doesn't work! Tried all combinations! I have many sites, I want to clear the cache for all the sites with one command! I wish that was possible here? What would we need to do? Get zones of all the sites and then do a loop on curl requests? 

This is the basic intent here for being able to use the -s — I was thinking of building the wrapper myself but you seem to be very active. We can collaborate.
